core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -57,6 +69,50 @@ public static void initPageVariables(JellyContext context) {
         context.setVariable("icons", icons);
     }
 
+    private String prependTitleIfRequired(String icon, String title) {
+        if (StringUtils.isNotBlank(title)) {
+            return "<span class=\"jenkins-visually-hidden\">" + title + "</span>" + icon;
```

Comment:
From what I can tell this means that icons will only have a title for screen readers now, is that intentional / what we want?

i.e. if I hover over build now nothing shows up, whereas if I hover over it on another Jenkins instance there's a title that pops up

Reply:
I'm using the `<title>` tag for the Font Awesome icons: https://github.com/jenkinsci/font-awesome-api-plugin/blob/master/src/main/resources/font-awesome/svg-icon.jelly

Reply:
The SVGs should be hidden due to aria-hidden, so I've added the title label in case the SVG represented something important that wasn't supported by text.